examples/.gitignore

```diff
@@ -1,9 +1,11 @@
 rdkafka_example
 rdkafka_performance
 rdkafka_example_cpp
-rdkafka_consumer_example
-rdkafka_consumer_example_cpp
+rdkafka_complex_consumer_example
```

Comment:
No ccloud example?

Reply:
The [ccloud examples](https://github.com/confluentinc/examples/blob/5.3.1-post/clients/cloud/README.md) are linked from the top-level [README](https://github.com/edenhill/librdkafka#usage-in-code) thanks to @ybyzek